test/installation/test.bats

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bats
```

Comment:
Remember that if you don't specify a [setup](https://github.com/clearlinux/swupd-client/blob/master/test/functional/testlib.bash#L2335) function a test environment is going to be created by default.
The same applies for the [teardown](https://github.com/clearlinux/swupd-client/blob/master/test/functional/testlib.bash#L2342) function.

If you want to save that extra time, just add an empty setup and teardown methods to your test:

```
test_setup() {

	# do nothing
	return

}

test_teardown() {

	# do nothing
	return

}
```